### PR TITLE
Add istio-pilot-agent-1.19 version stream

### PR DESCRIPTION
**Pull request by tcnghia**
* "istio-pilot-agent-1.19: new version stream for istio-pilot-agent"

This is based on istio-pilot-agent-1.18.yaml with updated tags, versions, epochs and commits.

#### For version bump PRs
- [x] The `epoch` field is reset to 0

### CODE REVIEW

File: istio-pilot-agent-1.19.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  - name: ${{package.name}}-compat
+    pipeline:
+      - runs: |
+          # link /usr/local/bin/pilot-agent -> /usr/bin/pilot-agent to match
+          # what the Istio Helm charts may expect.
+          mkdir -p ${{targets.subpkgdir}}/usr/local/bin
+          ln -sf /usr/bin/pilot-agent ${{targets.subpkgdir}}/usr/local/bin/pilot-agent
+    dependencies:
+      provides:
+        - istio-pilot-agent-compat=1.19.999
```
Comment (mattmoor) [inline istio-pilot-agent-1.19.yaml:52]: You don't need `.999` if this didn't exist in the pre version-stream package.

I think @vaikas also has a new syntax for this?

Comment (tcnghia) [reply istio-pilot-agent-1.19.yaml:52]: Thanks @vaikas  🙏  https://github.com/chainguard-dev/melange/blob/main/docs/BUILD-FILE.md#provides

Comment (tcnghia) [reply istio-pilot-agent-1.19.yaml:52]: Done

**Comment by vaikas** [comment]
Looking great to me. What I've been starting to do (and want to see about codifying into github actions tmw) is the tests that I run here (as an example):
https://github.com/wolfi-dev/os/pull/5191

tl;dr:
 * `make local-wolfi`
 * run `apk add <your package>
 
Then in another window to test updates:
GITHUB_TOKEN=<YOUR TOKEN> wolfictl check update  --override-version 0.0.1 YourNewPackage.yaml

**Review by vaikas — APPROVED** [review]
